mwdb/web/src/components/Settings/Views/GroupMembers.js

```diff
@@ -1,20 +1,155 @@
-        itemLinkClass={(user) => (
-            <UserBadge user={user} clickable basePath="/settings" />
-        )}
-        {...props}
-    />
-);
+import Pagination from "react-js-pagination";
+import { useViewAlert } from "@mwdb-web/commons/ui";
+import {
+    faUser,
+    faTrash,
+    faCrown,
+    faPlus,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Link } from "react-router-dom";
+import { Autocomplete, ConfirmationModal } from "@mwdb-web/commons/ui";
+
+function AddMemberForm({ newMemberItems, addMember }) {
+    const [newMember, setNewMember] = useState("");
+
+    return (
+        <div className="card">
+            <div className="card-body">
+                <Autocomplete
+                    value={newMember}
+                    getItemValue={(user) => user.login}
+                    items={newMemberItems.filter(
+                        (user) =>
+                            user.login
+                                .toLowerCase()
+                                .indexOf(newMember.toLowerCase()) !== -1
+                    )}
+                    onChange={(value) => setNewMember(value)}
+                    className="form-control"
+                    placeholder="User login"
+                />
+                <button
+                    className="btn btn-outline-success mt-2 mr-1"
+                    disabled={newMember.length === 0}
+                    onClick={() => {
+                        addMember(newMember);
+                        setNewMember("");
+                    }}
+                >
+                    <FontAwesomeIcon icon={faPlus} /> Add member
+                </button>
+            </div>
+        </div>
+    );
+}
+
+function MemberList({ members, admins, setAdminMembership, removeMember }) {
+    const [isModalOpen, setModalOpen] = useState(false);
+    const [modalSpec, setModalSpec] = useState({});
+
+    function setAdmin(member, membership) {
+        let message = `Are you sure to change admin group permissions for ${member}?`;
+
+        setModalSpec({
+            message,
+            action: () => {
+                setModalOpen(false);
+                setAdminMembership(member, membership);
+            },
+            buttonStyle: "bg-success",
+            confirmText: "Yes",
+        });
+        setModalOpen(true);
+    }
+
+    function removeMembership(member) {
+        let message = `Remove ${member} from this group`;
+
+        setModalSpec({
+            message,
+            action: () => {
+                setModalOpen(false);
+                removeMember(member);
+            },
+            confirmText: "Remove",
+        });
+        setModalOpen(true);
+    }
+
+    return (
+        <React.Fragment>
+            {members.map((member) => (
+                <tr>
+                    <th className="col">
+                        <span className="badge badge-info">{member.login}</span>
```

Comment:
I would use `UserBadge` from `@mwdb/commons/ui` here:
```suggestion
                        <UserBadge user={member} clickable basePath="/settings"/>
```

Reply:
![image](https://user-images.githubusercontent.com/8720367/168599687-be42eb9c-1169-4246-8a77-0483d4472e61.png)